Lib/traceback.py

```diff
@@ -449,6 +449,48 @@ def from_list(klass, a_list):
                 result.append(FrameSummary(filename, lineno, name, line=line))
         return result
 
+    def format_frame(self, frame):
```

Comment:
I just realized that this is actually a FrameSummary object rather than a frame.

Since we haven't released it yet, I think we should rename the function and its arg to format_frame_summary(frame_summary).   What do you think?

Reply:
I think is a good idea. What do you think @ammaraskar  @isidentical ?

Reply:
That sounds good to me, format frame summary seems more descriptive.

Reply:
Cool. I see the confusion between frame and frame_summary in local variable names predates this PR. I'll try to fix that as well.

Reply:
I've created bpo 45075 for this.